tests/robot-tests/tests/libs/admin_api.py

```diff
@@ -161,6 +162,40 @@ def user_adds_publication_role_to_user_via_api(
         f'Adding publication role to user API request failed with {response.status_code} and {response.text}'
 
 
+def user_removes_roles_from_prerelease_user_via_api(user_id: str) -> None:
+    response = admin_client.delete(
+        f'/api/user-management/user/{user_id}/release-role/pre-release'
+    )
+    assert response.status_code < 300, \
+        f'Removing release role from user API request failed with {response.status_code} and {response.text}'
+
+
+def user_removes_prerelease_user_roles_if_needed_via_api(user_emails: list) -> None:
+    allowed_users = [
+        'ees-prerelease1@education.gov.uk',
+        'ees-prerelease2@education.gov.uk',
+        'ees-prerelease3@education.gov.uk']
+
+    if len(set(user_emails).intersection(allowed_users)) < 2:
+        raise AssertionError('`User emails` must contain at least two allowed prerelease users')
+
+    user_ids = []
+    for user_email in user_emails:
+        try:
+            user_ids = [get_user_details_via_api(user_email)['id']]
+        except IndexError:
+            BuiltIn().log('User with email {} does not exist'.format(user_email))
+
+        try:
+            result = [user_removes_roles_from_prerelease_user_via_api(user_id) for user_id in user_ids]
+            assert(all(result))
+
+        except AssertionError:
+            BuiltIn().log(f'Failed to remove roles from user {user_email}')
+
+    BuiltIn().log(f'All userReleaseRoles & userPublicationRoles removed from prerelease user: {user_email}')
```

Comment:
Again, I wouldn't expect a prerelease user to have any publication-level roles, or else they shouldn't be a prerelease user :)

Reply:
I've renamed this to `BuiltIn().log(f'All userReleaseRoles & userPublicationRoles reset for user: {user_email}')` now